pocs/camera/fli.py

```diff
@@ -45,20 +46,22 @@ def __init__(self,
         # Create an instance of the FLI Driver interface
         self._FLIDriver = libfli.FLIDriver(library_path)
 
+        logger = get_root_logger()
```

Comment:
It's just because I don't want to call `super().__init__()` until later, once I have the correct values in `self._serial_number` and `self.port`, so that the log entries generated by `super().__init__()` don't use the placeholder default values for those.  Until I make that call `self.logger` doesn't exist so to enable logging in the port finding section I need to use `get_root_logger()`.

A bit of a faff just to avoid 'XXXXXX' appearing in the logs, but it works.